chain/chain/src/chain.rs

```diff
@@ -311,6 +311,16 @@ impl OrphanBlockPool {
         res
     }
 
+    pub fn list_orphans_by_height(&self) -> Vec<BlockStatusView> {
+        let mut rtn = Vec::new();
+        for (height, orphans) in &self.height_idx {
```

Comment:
nit: you can probably do it in more 'rusty' way -- wtih .iter().map(...) and collect

Reply:
For sure.